docs/installation.rst

```diff
@@ -42,6 +42,8 @@ Add the following to the `urlpatterns` in your `urls.py` to expose the webhook e
 
     url(r'^payments/', include('djstripe.urls', namespace="djstripe")),
 
+Then tell stripe about the webhook (Stripe webhook docs can be found `here <https://stripe.com/docs/webhooks>`_) using the full url for wherever you put the the djstrip urls, e.g. ``yourwebsite.com/payments/webhook``.
```

Comment:
"using the full url of your endpoint from the urls.py step above (e.g. ``https://yourwebsite.com/payments/webhook``)"